include/hc.hpp

```diff
@@ -23,6 +23,7 @@
 
 #include <hsa_atomic.h>
 #include <kalmar_cpu_launch.h>
+#include <hsa_ext_amd.h>
```

Comment:
@facao Could you remove this line from hc.hpp ? As there is no HSA runtime API directly used in hc.hpp we should remove it to completely remove the dependency.